peerconnection.go

```diff
@@ -2268,8 +2291,11 @@ func (pc *PeerConnection) startTransports(iceRole ICERole, dtlsRole DTLSRole, re
 	}
 
 	pc.dtlsTransport.internalOnCloseHandler = func() {
-		pc.log.Info("Closing PeerConnection from DTLS CloseNotify")
+		if pc.isClosed.get() {
```

Comment:
Is this getting fired multiple times right now?

Reply:
It doesn't, but it can get fired from a DTLS closenotify while Close is happening, if the timing is right.